frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '@console/internal/components/dashboard/dashboard-card';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { BucketsItem, bucketsType } from './buckets-card-row';
+import './buckets-card.scss';
+
+enum BucketsCardQueries {
+  BUCKETS_COUNT = 'NooBaa_num_buckets',
+  BUCKET_OBJECTS_COUNT = 'NooBaa_num_objects',
+  BUCKET_CLAIMS_COUNT = 'NooBaa_num_buckets_claims',
+  BUCKET_CLAIMS_OBJECTS_COUNT = 'NooBaa_num_objects_buckets_claims',
+  UNHEALTHY_BUCKETS = 'NooBaa_num_unhealthy_buckets',
+  UNHEALTHY_BUCKETS_CLAIMS = '',
```

Comment:
what is this empty query ?

Reply:
thats why its in WIP witing for the query from nooba guys